src/test/groovy/edu/ie3/util/io/FileIOUtilsTest.groovy

```diff
@@ -0,0 +1,385 @@
+/*
+ * © 2020. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+package edu.ie3.util.io
+
+import edu.ie3.util.exceptions.FileException
+import org.apache.commons.compress.archivers.ArchiveEntry
+import org.apache.commons.io.FilenameUtils
+import spock.lang.Shared
+import spock.lang.Specification
+
+import java.nio.file.Files
+import java.nio.file.Path
+import java.nio.file.Paths
+import java.util.concurrent.CompletionException
+import java.util.stream.Collectors
+
+class FileIOUtilsTest extends Specification {
+	@Shared
+	Path tmpDirectory
+
+	def setup() {
+		tmpDirectory = Files.createTempDirectory("psdm_fileio utils")
+	}
+
+	def cleanup() {
+		FileIOUtils.deleteRecursively(tmpDirectory)
+	}
+
+	def "The fileio utils throws an exception, if the input path is null"() {
```

Comment:
Please adapt the descriptions or the test class name, as it currently is not too descriptive. E.g. you cannot see at execution, with which operation (compressing) the exception is meant to be thrown.

Reply:
Okay, I will elaborate the descriptions.